guides/common/assembly_importing-content.adoc

```diff
@@ -170,6 +170,8 @@ endif::[]
 
 include::modules/proc_enabling-red-hat-repositories.adoc[leveloffset=+1]
 
+include::modules/ref_required-red-hat-repositories.adoc[leveloffset=+2]
```

Comment:
Not sure if this should go to an appendix. WDYT?

Reply:
That might help, yes. What would you think about converting the contents into a table too?

Or... you could keep it in the procedure module because these are prerequisites, right? Things that users _always_ need when following the procedure? So they might not appreciate being redirected to a different section, only to have to return immediately after. But I understand your concern about the list being too long now.

It might be a good use case for tabs, too: https://docs.theforeman.org/nightly/Installing_Server/index-foreman-deb.html#configuring-repositories_foreman Tabs would help keep the list more compact and then it would work better in its original place.

Reply:
I don't like the idea of a table for this case.
No, they aren't formal prerequisites. The procedure is generic for enabling repos. The reference tells the user which repos they need for what purposes so that they know what repos they should enable.

How are tabs handled downstream?